packages/lambda-xyz/src/tile.set.ts

```diff
@@ -70,14 +70,8 @@ export class TileSet {
         return true;
     }
 
-    *allImagery(): Generator<{ rule: TileMetadataImageRule; imagery: TileMetadataImageryRecord }> {
-        const rules = Aws.tileMetadata.TileSet.rules(this.tileSet);
-        for (const rule of rules) {
-            const imagery = this.imagery.get(rule.id);
-            if (imagery == null) throw new Error(`Unable to find imagery for key: ${rule.id}`);
-
-            yield { rule, imagery };
-        }
+    *allImagery(): Generator<TileSetRuleImagery> {
```

Comment:
I think we can just remove this, and expose the `imagery` again, the reason it was made private was because it was no longer sorted.